### PR TITLE
update names and links

### LINKED ISSUE #14: update links
update links between slides

### PR DESCRIPTION
**Pull request by clarkliming**
close #14

### CODE REVIEW

File: slides/05_collaboration.qmd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "5 Version Control & Collaboration"
-author: "Kevin"
+author: "Shuang"
```
Comment (shajoezhu) [inline slides/05_collaboration.qmd]: I think we should add date too

File: slides/03_workflow.qmd [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "3 An R Package Engineering Workflow"
-author: "Friedrich"
+author: "Daniel"
```
Comment (shajoezhu) [inline slides/03_workflow.qmd:3]: I think we should add date too

File: slides/01_intro.qmd [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "1 Introduction"
-author: "Daniel, Friedrich, Kevin"
+author: "Daniel"
```
Comment (shajoezhu) [inline slides/01_intro.qmd]: I think we should add date too

**Review by shajoezhu — COMMENTED** [review]
I think we should add date too

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**